test/libyul/ObjectParser.cpp

```diff
@@ -74,7 +74,7 @@ pair<bool, ErrorList> parse(string const& _source)
 	return {false, {}};
 }
 
-optional<Error> parseAndReturnFirstError(string const& _source, bool _allowWarningsAndInfos = true)
+std::optional<Error> parseAndReturnFirstError(string const& _source, bool _allowWarningsAndInfos = true)
```

Comment:
We do not add these in .cpp files where we have `using namespace std`:
```suggestion
optional<Error> parseAndReturnFirstError(string const& _source, bool _allowWarningsAndInfos = true)
```